README.md

```diff
@@ -474,6 +474,49 @@ $ rustup target add i686-pc-windows-gnu
 [GCC toolchain]: https://gcc.gnu.org/
 [MinGW/MSYS2 toolchain]: https://msys2.github.io/
 
+### VS Build Tools Caveats
+
+When selecting the Build Tools only instalation, the Rust tooling will work as
+expected, but you might face a couple of issues when compiling `*-sys` 
+dependencies.
+
+`*-sys` dependencies often link to C/C++ libraries as part of their build 
+system, and for that, a set of environment variables and paths should be setup.
+
+Selecting the Build Tools instalation instead of the complete Visual Studio 
```

Comment:
Perhaps "Selecting only the _Build tools_ instead of a complete installation of Visual Studio..." instead?